app/data/helpers/generators.js

```diff
@@ -0,0 +1,72 @@
+
+
+const faker = require('faker')
+faker.locale = 'en_GB'
+
+exports.firstName = (sex) => {
+  if(sex === 'male') {
+    return faker.helpers.randomize([
+      'Tony',
+      'Thor',
+      'Henry',
+      'Robert',
+      'Steve',
+      'Clinton',
+      'Pietro',
+      'Jacques',
+      'Harry',
+      'Luke',
+      'Victor',
+      'Samuel'
+    ])
+  } else {
+    return faker.helpers.randomize([
+      'Janet',
+      'Wanda',
+      'Natasha',
+      'Heather',
+      'Patricia',
+      'Carol',
+      'Jennifer',
+      'Monica',
+      'Barbara',
+      'Bonita',
+      'Susan',
+      'Julia'
+    ])
+  }
+}
+
+exports.lastName = (sex) => {
+  if(sex === 'Male') {
+    return faker.helpers.randomize([
+      'Stark',
```

Comment:
You're missing:

- Hogan
- Octavius
- Quill
- Schultz
- Tivan

at least...!

Reply:
There's also

- Foster